### PR TITLE
adds field new fieldnames used by adyen

### LINKED ISSUE #13: Update to new Adyen CSS selectors
Adyen changes there CC form fiels IDs to be prefix with 'card.'. Skinbuilder should reflect this.

But instead of updating the HTML, why not introducing an option where each skin has it's own skeleton templates? This
way the correct payment methods are shown too.

Basic Flow to get the skeletion code:
- upload an empty skin
- load skin, scrape code, split into parts
- save as skeleton templates
- make sure  skeleton templates don't get added to .zip file

### PR DESCRIPTION
**Pull request by jigfox**
also adds more defensive i18n check so test are working again and create the option tags for year dynamically to reflect the current year

fixes priithaamer/adyen-skinbuilder#13

### CODE REVIEW

File: lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb [hunk 1]
```diff
@@ -461,26 +461,19 @@
...
             </select>
             </div>
         </td>
       </tr>
       <tr>
         <td><div>Card expiry date</div></td><td>
-        <div class="fieldDiv">
-        <select class="inputField" name="expiryMonth" id="expiryMonth" size="1">
+        <div class="fieldDiv" id="card.expiryContainer">
+        <select class="inputField" name="card.expiryMonth" id="card.expiryMonth" size="1">
```
Comment (rngtng) [inline lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb:476]: could you indent please

**Comment by rngtng** [comment]
cool thanks...

File: lib/adyen-skinbuilder/server.rb [hunk 2]
```diff
@@ -65,10 +69,10 @@ def render_skin(skin)
       helpers Helper::Render, Helper::Adyen
 
       before do
-        if settings.i18n_path
+        if settings.respond_to?(:i18n_path) && settings.i18n_path
```
Comment (rngtng) [inline lib/adyen-skinbuilder/server.rb:72]: what was the problem here?

File: lib/adyen-skinbuilder/server.rb [hunk 3]
```diff
@@ -15,16 +15,20 @@ class Server < Sinatra::Base
...
 
       set :views, "#{dir}/server/views"
-      
+
       # method will be overwritten by _vegas_ if skin directory given
       def self.skins_directory
         File.expand_path(".")
       end
 
+      def self.skins_directory=(dir)
```
Comment (rngtng) [inline lib/adyen-skinbuilder/server.rb:24]: for what do you need this for?

File: lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb [hunk 4]
```diff
@@ -461,26 +461,19 @@
...
-            <option value="2009">2009</option>
-            <option value="2008">2008</option>
-            <option value="2007">2007</option>
-            <option value="2006">2006</option>
-            <option value="2005">2005</option>
-            <option value="2004">2004</option>
-            <option value="2003">2003</option>
-            <option value="2002">2002</option>
+            <% 10.times do |i| %>
+              <option value="<%= year = Date.today.year - i %>"><%= year %></option>
```
Comment (rngtng) [inline lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb:467]: still not perfect, but what about

``` erb
<%= '<option value="%s">%s</option>' % ([Date.today.year - i] * 2) %>
```

File: lib/adyen-skinbuilder/server.rb (hunk 2 above)
Comment (jigfox) [inline lib/adyen-skinbuilder/server.rb:72]: undefiened method `i18n_path` that's why the tests wasn't green

File: lib/adyen-skinbuilder/server.rb (hunk 3 above)
Comment (jigfox) [inline lib/adyen-skinbuilder/server.rb:24]: I need this for me, so I can run it in my own sinatra app and configure a path where to look for the skin

File: lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb (hunk 4 above)
Comment (jigfox) [inline lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb:467]: would work, too

File: lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb (hunk 1 above)
Comment (jigfox) [inline lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb:476]: yes i could, but this will change the whole file, and would make the diff a lot more difficult to read

File: lib/adyen-skinbuilder/server.rb (hunk 2 above)
Comment (rngtng) [inline lib/adyen-skinbuilder/server.rb:72]: ah cool thx

File: lib/adyen-skinbuilder/server.rb (hunk 3 above)
Comment (rngtng) [inline lib/adyen-skinbuilder/server.rb:24]: ah ok, I'd preferably see this change in a separate commit.

File: lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb (hunk 1 above)
Comment (rngtng) [inline lib/adyen-skinbuilder/server/views/_adyen_payment_fields.html.erb:476]: as long it's a separate commit I don't see any problem with that...

File: .travis.yml [hunk 5]
```diff
@@ -3,3 +3,4 @@ rvm:
   - 1.8.7
   - 1.9.2
   - 1.9.3
+  - 2.0.0
```
Comment (rngtng) [inline .travis.yml:6]: wait this is not part of this PR right? pls move it to a new one..
